setup.py

```diff
@@ -67,6 +67,7 @@ def find_version(*file_paths):
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
-        "Programming Language :: Python :: 3.11"
+        "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: 3.12"
     ],
 )
```

Comment:
I think line 30 in `setup.cfg` can also be removed since only py >=3.4 is supported: `'mock;python_version<"3.3"',`

Reply:
ps. dropping support for python 3.4, 3.5 and 3.6 might also be a good idea since they're all EOL (even 3.7 is now).

Reply:
Cheers. Can probably drop that entire `test_deps` block as we replaced `nose` with `pytest` during the transition to Github actions.